content/influxdb/v2.0/reference/flux/stdlib/monitor/notify.md

```diff
@@ -29,26 +29,56 @@ monitor.notify(
 ## Parameters
 
 ### endpoint
-A function that constructs and sends the notification to an endpoint.
+({{< req >}}) A function that constructs and sends the notification to an endpoint.
 
 _**Data type:** Function_
 
 ### data
-Data to append to the output.
-**InfluxDB populates notification data.**
+({{< req >}}) Notification data to append to the output.
+This data specifies which notification rule and notification endpoint to associate
+with the sent notification.
 
 _**Data type:** Record_
 
+The data record must contain the following fields:
+
+- _notification_rule_id
+- _notification_rule_name
+- _notification_endpoint_id
+- _notification_endpoint_name
+
+The InfluxDB monitoring and alerting system uses `monitor.notify()` to store
+information about sent notifications and automatically assigns these values.
+If writing a custom notification [task](/influxdb/v2.0/process-data/),
+use **unique arbitrary values**.
+
 ## Examples
```

Comment:
Shall we add examples for pagerduty/http?

Reply:
I added a PagerDuty example, but they're all essentially the same. The only thing that would change between them is the endpoint configuration.